ui/src/index.server.jsx

```diff
@@ -26,7 +27,13 @@ export const render = (props: Object) => {
     return `
         <!DOCTYPE html>
         <html lang="en">
-            ${head(props, styletron.getStylesheetsHtml())}
+            ${head(
+                props,
+                [
+                    styletron.getStylesheetsHtml(),
+                    `<style expensive-css>${extractCritical(body).css}</style>`,
```

Comment:
What is the `expensive-css` attribute for? Is it simply flagging up this style element for developers / archaeologists?

Reply:
yeah exactly that